VideoWeb/VideoWeb/ClientApp/.prettierignore

```diff
@@ -1,2 +1 @@
 *.component.html
-*.orig
```

Comment:
You don't want prettier to be running on .orig files. they're files left over after a merge